### DISCUSSION TITLE
Remove Metronome From "Muted" Mod

### DISCUSSION THREAD

**Discussion by mooshiros** [ppy/osu#29668] [Ideas]
Having a metronome makes the game significantly easier, even easier than actually having the song. Muted seems like it's supposed to test the player's ability to keep their own rhythm, but it genuinely makes it far easier to get good accuracy and is overpowered.

EDIT: Alternatively if "fun" mods didn't submit to the leaderboard/give score (which they shouldn't imo) this wouldn't be an issue

**Comment by MeerukYang** [comment]
can confirm that only a full ticking metronome can in some case make it easier to catch the beat. But your suggestion is impractical because osu! is anyhow a **rhythm** game where you play according to the song or beat. removing metronome makes beatmap completely in silence, making the game strange like u certainly cannot feel the rhythm without a actual thythm.
My suggestion is to make metronome tick once at the first baet every bar; since a lot of songs are written in 4/4 common time or 3/4 triple meter, the once-per-bar tick is both challenging but enough to hint the players.

**Comment by shiumano** [comment]
Did you know that you can turn off the metronome tick in Muted from the Mod customization panel?

**Reply by mooshiros** [reply]
Yes, but my whole point is that the mod gives an unfair advantage, so why would anyone turn the metronome off?

**Reply by peppy** [reply]
If you want a challenge?

**Reply by mooshiros** [reply]
@peppy Well sure but that's not my point, my point is that the mod is abusable. Of course if you're intent is to have a challenge then you can turn it off, but my issue with the metronome being an option at all is that it's abusable to get good accuracy (and thus better scores) on maps that would otherwise be hard to get good accuracy on. As just a personal example, on the Aug 31 daily challenge (a map which, in my and many other opinions, follows the song very weirdly), my accuracy went from 93% to 96% immediately by turning on muted with the metronome. Something that gives that sort of score increase for free just isn't fair imo. I also know I'm not the only one who used the muted mod with metronome on that daily challenge.

**Reply by Tenexxt** [reply]
Muted mod isn't a diff increase mod, its a FUN mod. You aren't supposed to be rewarded for playing with it, since its supposed to be a "Oh Muted mod? Yea its fun to use from time to time" and not a pp/score farm mod.
Sometimes i also use Muted to just play with the Mentonome because i have a hard time catching the beat (no, not like ctb, tho i also play it) and perfer to play with music off and with the metronome.
Also as for you getting a better score with MU on, did you consider that if you play the same map more you will get better at playing it, and maybe its not the mod being "OP"?
Also how would MU's metronome be op in the first place? If people wanted to play with a metronome they would find a way, so why call it an advantage if its literally a matter of preference and the song?

**Reply by mooshiros** [reply]
@Tenexxt 
>You aren't supposed to be rewarded for playing with it, since its supposed to be a "Oh Muted mod? Yea its fun to use from time to time" and not a pp/score farm mod.

Scores set with the mod get submitted to the leaderboard and you gain score from them though.

>Sometimes i also use Muted to just play with the Mentonome because i have a hard time catching the beat (no, not like ctb, tho i also play it) and perfer to play with music off and with the metronome.

Thank you for proving my point exactly, you're abusing the mod to set scores that you don't have the rhythm sense to set legitimately. This wouldn't be an issue if the scores with the mod didn't submit to the leaderboard, but they do.

>Also as for you getting a better score with MU on, did you consider that if you play the same map more you will get better at playing it, and maybe its not the mod being "OP"?

Ah yes so when I play try the map 10 times without the mod and consistently get 92-93%, then immediately jump to 95-97% runs when I turn on the mod that's clearly not because of the metronome. And again I'm not the only one who did this and found the metronome to be immensely helpful.

>Also how would MU's metronome be op in the first place?

I literally explained this? It helps you keep on rhythm for songs/maps that would otherwise be hard to keep rhythm on. It eliminates a big part of the challenge for some finger control-y maps.

>If people wanted to play with a metronome they would find a way

And? Just because it's possible to use external tools to eliminate the challenge from certain maps (by the way I 100% think using a metronome should be a punishable offense) doesn't mean the game should enable that behavior. Besides, having an external metronome be synced up with the map properly would be a challenge, not to mention the fact that any timing changes would be near impossible to adjust for (unless you have a program set up that switches the metronome at certain points, which again seems very much like it should be punishable). This whole argument is like saying people can place markers on their screen to cheese flashlight plays so why don't we add skinnable markers that are visible when flashlight is on. It's just a bad argument.

>why call it an advantage if its literally a matter of preference and the song?

Because it literally is an advantage for certain songs. You say it's a matter of preference but this is a rhythm game, where you hit to the beat of the song, having a metronome defeats a huge aspect of the game. Yes it's a "fun mod," but the scores with it submit to the leaderboard which makes it abusable. It would be fine if it gave you no benefit, but it literally does.

**Reply by Tenexxt** [reply]
> Ah yes so when I play try the map 10 times without the mod and consistently get 92-93%, then immediately jump to 95-97% runs when I turn on the mod that's clearly not because of the metronome. And again I'm not the only one who did this and found the metronome to be immensely helpful.

And there is a even bigger amount of people (like me) that didn't play with MU that day, and often don't. Its a matter of **preference** and **how well someone can catch the songs beat**. There will be people who will always play with MU because they like the metronome, and there will be people who will never touch it because **they prefer to play without it**.
As a pianist its not hard for me to get a songs rhythm but sometimes i like to play with the metronome to just have fun and just to have some difference in my playthrough.

> It helps you keep on rhythm for songs/maps that would otherwise be hard to keep rhythm on. It eliminates a big part of the challenge for some finger control-y maps.

Again, highly specific argument that can be easily countered with the fact that not only this would be rare to encounter since most mappers prefer to map simple songs, and even if there is a map with a complicated rhythm, there is also a even bigger chance that the metronome will actually be your doom, since it works on 1/1 ticks and since the rhythm is complicated, you will most likelly get a note that will also get a weird offset in the game so 1/1 ticks will actually be annoying is some cases - just like in some cases the metronome will help some people and doom others every single time they touch it.

>  This whole argument is like saying people can place markers on their screen to cheese flashlight plays so why don't we add skinnable markers that are visible when flashlight is on. It's just a bad argument.

Yea, exept FL gives you not only score and overall pp bonus, it also has its own "bonus" pp reward, unlike MU where you play like normal without any score and pp bonuses. The gameplay isn't touched, so no reason to unrank it imo

**Reply by mooshiros** [reply]
@Tenexxt 
>Its a matter of preference and how well someone can catch the songs beat.

Bro, what? "how well someone can catch the song's beat" isn't a matter of preference, it's a basic necessity for rhythm games. If you have trouble "catching a song's beat," you should be doing bad, not have an option to artificially make it easier to stay on beat.

>As a pianist its not hard for me to get a songs rhythm but sometimes i like to play with the metronome to just have fun and just to have some difference in my playthrough.

I'm a pianist too, so let me ask you this: when you perform, do you perform with a metronome? Sure you can use a metronome in practice, but submitting scores with it shouldn't be a thing, just like performing with a metronome isn't a thing either.

>Again, highly specific argument that can be easily countered with the fact that not only this would be rare to encounter since most mappers prefer to map simple songs

Yeah, and this highly specific case is the case in which MU is abusable, as in, what my whole point is about. The metronome is irrelevant for most maps, and for some maps it provides an unfair advantage. That's my whole point.

>there is also a even bigger chance that the metronome will actually be your doom, since it works on 1/1 ticks and since the rhythm is complicated, you will most likelly get a note that will also get a weird offset in the game so 1/1 ticks will actually be annoying is some cases - just like in some cases the metronome will help some people and doom others every single time they touch it.

This is a strawman argument. The fact that there are complex maps where the metronome makes it harder doesn't change the fact that there are complex maps where the metronome makes it unfairly easier with no score punishment.

>Yea, exept FL gives you not only score and overall pp bonus, it also has its own "bonus" pp reward, unlike MU where you play like normal without any score and pp bonuses. The gameplay isn't touched, so no reason to unrank it imo

Sure there's no score bonus for MU, but it's easier on some maps to get better accuracy (and thus better score) with it. And the gameplay is touched, I don't know what you're talking about, a huge part of the gameplay is using the song to stay on beat. Metronome makes it easier to stay on beat, and is thus an unfair gameplay advantage. That's like saying FL or EZ should give 1.0 score multiplier and not affect PP at all because "the gameplay isn't touched." Reading matters, and the audio side of that matters as much as the visual side.

Also, why are you so adamant about the mod remaining ranked (for the leaderboards) if you don't intend to abuse it? If you just want to turn off the music, you can do that with the volume controls, just like you could on stable. You already admitted that it helps you stay on beat on certain maps (which is, again, my entire point), so it seems to me like you're only arguing for it to keep being ranked out of self-interest rather than looking at it objectively.

**Reply by Tenexxt** [reply]
While I'd prefer for the mod to stay ranked, i have no bigger problem with the metronome trigger making the mod unranked (If this is what you mean, change The title because its giving mixed signals btw)

**The metronome should stay**, but it can make the mod unranked if necessary (i use the mod for fun and not scores anyway lol)

**Reply by shiumano** [reply]
I would like to leave the metronomtic off by default, and when the metronome is turned on, set the score multiplier to 0.8x (well, about the same as Synesthesia?).and set it to Unranked.
As a matter of fact, I don't like the metronome tick at all.

**Reply by Tenexxt** [reply]
Yea that seems balanced imo

**Reply by mooshiros** [reply]
I'd like to reopen this discussion now that muted is ranked and you can gain pp from it. Now the fact that the mod is abusable to more easily get high accuracy on rhythmically difficult maps is an even bigger issue than when I made my initial post. I really don't think muted with metronome should give pp (or score) for this reason.
